src/Store.js

```diff
@@ -8,8 +8,8 @@ import { persistStore, persistReducer } from 'redux-persist';
 import { applyMiddleware, createStore } from 'redux';
 import thunk from 'redux-thunk';
 
+import { navigationMiddleware } from './navigation';
```

Comment:
just put this in navigation folder - thought it was better but now not so sure

Reply:
I prefer it in `./navigation`